makemkvchapters

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+#makemkvchapters creates ordered chapters in a matroska file
+SCRIPTDIR=$(dirname "${0}")
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+unset DEPENDENCIES
+DEPENDENCIES=(mkvpropedit)
+_initialize_make
+
+while [ "${*}" != "" ] ; do
+    INPUT="${1}"
+    shift
+    if [ -f "${INPUT}" ] ; then
+        MKVCHAPTERS_XML=$(_maketemp)
+        MKVTAGS_XML=$(_maketemp)
+
+_report "For the Introduction"
+_report "Start and end times must be in HH:MM:SS.mmm or S.mmm format. Note mmm is milliseconds and not frames."
+_report -qn "Enter the start time or 'q' to quit: "
+read -e intimeI
+[[ "${intimeI}" = "q" ]] && exit 0
```

Comment:
I suggest to use (also below) the test also on the capital letter
`[[ "${intimeI}" = [qQ] ]]`
… as IRL this is an often made mistyping.